decentraland/unity-renderer
# Fix: render all empty parcels with same color

## What does this PR change?

Fix #2416 
Increases the range of the reflection probe to include all nearby parcels in the dynamic baking 

...

## How to test the changes?

1. Go to: https://play.decentraland.zone/?renderer-branch=fix/floor-color
2. Go to an isolated area like -66,-86
3. Verify that the colour of the ground in the nearby parcels is always the same

## Our Code Review Standards

https://github.com/decentraland/unity-renderer/blob/master/docs/code-review-standards.md
